### PR TITLE
Merge sync-js branch

### PR DESCRIPTION
**Pull request by vidurb**

### CODE REVIEW

File: assets/package.json.default [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+    "name": "miranj/craft-build",
+    "version": "0.0.1",
+    "main": "gulp.js",
+    "author": "Miranj",
+    "devDependencies": {
+        "autoprefixer": "^9.4.7",
+        "fontfaceobserver": "^2.1.0",
+        "fontloader": "gist:4e2c2cba3affe643ff18bd4eaf869678",
```
Comment (rungta) [inline assets/package.json.default:10]: Nice! Did not know that npm can pull in from gists as well.

**Comment by rungta** [comment]
Any reason the file is named `package.json.default`?

File: assets/package.json.default (hunk 1 above)
Comment (vidurb) [reply assets/package.json.default:10]: I had to fork your gist to add a package.json to enable that.

**Comment by vidurb** [comment]
What's your opinion?

**Comment by rungta** [comment]
Isn’t everything in this request also in #19? I’m missing the point of the duplication. Should I just close this and focus on #19?

**Comment by vidurb** [comment]
If that's what you think is best. 
I can move the commits from here to that branch. 
There is no 'duplication' that I can see.

**Comment by vidurb** [comment]
Also, I can refactor this branch and sync-build to move the JS utilities that were added in sync-build to this branch in order to further separate concerns.

**Pull request merged**